x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/connector/syncjob/action/UpdateConnectorSyncJobIngestionStatsAction.java

```diff
@@ -57,6 +57,7 @@ public static class Request extends ConnectorSyncJobActionRequest implements ToX
         private final Long indexedDocumentVolume;
         private final Long totalDocumentCount;
         private final Instant lastSeen;
+        private final Object metadata;
```

Comment:
Is using `Object` a standard thing to do in our APIs for "JSON Objects?" Feels too generic, since `Object` is the base class of all things Java. I would have expected `Map<String, String>` or `JsonObject` or something less vague. But that might just be me not being familiar with established ES patterns.